tests/integration/component_provider/python/package/yaml/Pulumi.yaml

```
@@ -0,0 +1,14 @@
+name: python-component-provider-yaml
+description: Using a component provider written as a Python package from YAML
+runtime: yaml
+plugins:
+  providers:
+    - name: provider
+      path: ../provider
```

```suggestion
packages:
  provider: ../provider
```

I think we could use the new style packages section here.